ControlRoomApplication/ControlRoomApplication/Controllers/ControlRoomController/RadioTelescopeControllerManagementThread.cs

```diff
@@ -163,21 +173,28 @@ public void InterruptOnce()
         private void SpinRoutine()
         {
             bool KeepAlive = KeepThreadAlive;
-            
+            //Appointment OldAppointment = new Appointment();
```

Comment:
We prolly want to remove this temporary comment just to keep the code nice and clean!